src/Examples/Flowsheets/power_generation/ngcc/ngcc_soec.ipynb

```diff
@@ -453,17 +396,18 @@
    "metadata": {},
    "outputs": [
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Extreme Jacobian entries:\n",
-      "Badly scaled variables:\n",
-      "2022-11-17 10:20:42 [WARNING] idaes.core.util.scaling: Nonsquare Jacobian using pseudo inverse\n",
-      "Jacobian Condition Number: 9.62e+09\n"
-     ]
+     "data": {
+      "text/plain": [
+       "'\\njac, nlp = iscale.get_jacobian(m, scaled=True)\\nprint(\"Extreme Jacobian entries:\")\\nfor i in iscale.extreme_jacobian_entries(jac=jac, nlp=nlp, large=1000, small=0):\\n    print(f\"    {i[0]:.2e}, [{i[1]}, {i[2]}]\")\\nprint(\"Badly scaled variables:\")\\nfor v, sv in iscale.badly_scaled_var_generator(\\n    m, large=1e3, small=1e-6, zero=1e-12\\n):\\n    print(f\"    {v} -- {sv} -- {iscale.get_scaling_factor(v)}\")\\nprint(f\"Jacobian Condition Number: {iscale.jacobian_cond(jac=jac):.2e}\")\\n'"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
+    "\"\"\"\n",
     "jac, nlp = iscale.get_jacobian(m, scaled=True)\n",
```

Comment:
This output looks a bit strange on GitHub, does it parse correctly locally?

Reply:
I commented out the block of code to check scaling using `""" ... """`.  It looks like that because the " are in another string so there are escape characters.  It renders fine.